src/nwbinspector/checks/images.py

```diff
@@ -29,3 +30,11 @@ def check_order_of_images_len(images: Images):
                 message=f"Length of order_of_images ({len(images.order_of_images)}) does not match the number of "
                 f"images ({len(images.images)})."
             )
+
+    @register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=IndexSeries)
+    def check_index_series_points_to_image(index_series: IndexSeries):
+        if index_series.indexed_timeseries is not None:
+            return InspectorMessage(
+                message="Pointing an IndexSeries to a TimeSeries is is deprecated. Please point to an Images container "
```

Comment:
Technically the schema says it 'will be' deprecated. https://github.com/NeurodataWithoutBorders/nwb-schema/blob/dev/core/nwb.image.yaml#L215-L216

Should we mirror that language or act as if it already has been?